### PR TITLE
BaSP-MR-30: Test admin put delete

### PR DESCRIPTION
**Pull request by IgnacioGalceran**
Added data in Admin.js file in seed folder.
Added tests to put and delete endpoints.

### CODE REVIEW

File: src/tests/admins.test.js [hunk 1]
```diff
@@ -1 +1,90 @@
...
+  test('should return status 200 admin updated', async () => {
+    const response = await request(app).put('/api/admins/6465125fb6b5507c22ad8dce').send(mockAdmin);
+    expect(response.body.message).toBe('Admin updated');
+    expect(typeof response.body.data).toBe('object');
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+});
+
+describe('PUT /api/admins', () => {
```
Comment (Micaela-Rossi) [inline src/tests/admins.test.js:67]: Here must said DELETE instead of PUT

Comment (Eliezer-Alberto) [reply src/tests/admins.test.js:67]: Apart from that, It is all fine!..

File: src/tests/admins.test.js [hunk 2]
```diff
@@ -1 +1,90 @@
...
+    const response = await request(app).put('/api/admins/6465125fb6b5507c22ad8dcd').send({ dni: 33323343 });
+    expect(response.body.message).toBe('There is another admin with that data.');
+    expect(typeof response.body.data).toBe('object');
+    expect(response.status).toBe(400);
+    expect(response.error).toBeTruthy();
+  });
+  test('should return status 400 email already exists', async () => {
+    const response = await request(app).put('/api/admins/6465125fb6b5507c22ad8dcd').send({ firstName: 'admin2', email: 'testing@gmail.com' });
+    expect(response.body.message).toBe('There is another admin with that data.');
+    expect(typeof response.body.data).toBe('object');
```
Comment (nailaghigo) [inline src/tests/admins.test.js:33]: Are we sure that we are testing that our data is empty? I think we could use the following example 
`expect(response.body.data).toEqual({})`

Check if we have to replace something similar throughout the code

File: src/tests/admins.test.js [hunk 3]
```diff
@@ -1 +1,90 @@
...
+    const response = await request(app).put(`/api/admins/${idInvalid}`).send(mockAdmin);
+    expect(response.body.message).toBe('The ID is not valid');
+    expect(response.body.data).toBe(idInvalid);
+    expect(response.status).toBe(400);
+    expect(response.error).toBeTruthy();
+  });
+  test('should return status 404 admin not found', async () => {
+    const response = await request(app).put('/api/admins/6264125fb6b5507c22ad8dce').send(mockAdmin);
+    expect(response.body.message).toBe('Admin with the id (6264125fb6b5507c22ad8dce) was not found.');
+    expect(response.body.data).toBe(undefined);
```
Comment (nailaghigo) [inline src/tests/admins.test.js:47]: We could use `.toBeUndefined()` jest matcher here.

**Review by nailaghigo — CHANGES_REQUESTED** [review]
🎸

File: src/tests/admins.test.js [hunk 4]
```diff
@@ -1 +1,90 @@
...
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+});
+
+describe('PUT /api/admins', () => {
+  test('should return status 404 id not found', async () => {
+    const response = await request(app).delete('/api/admins/6465113fb6b5507c22ad8dcd').send();
+    expect(response.body.message).toBe('Admin with ID (6465113fb6b5507c22ad8dcd) was not found');
+    expect(response.body.data).toBe(undefined);
```
Comment (Eliezer-Alberto) [inline src/tests/admins.test.js:71]: Could it be used `.toBeUndefined();` jest matcher here..?

**Review by Eliezer-Alberto — APPROVED** [review]
I think It's all fine!..

**Review by nailaghigo — APPROVED** [review]
🚀

**Pull request merged**